### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mh-cbon**
suggestion about #73

### CODE REVIEW

**Comment by TheAlphaNerd** [comment]
I'm not 100% about adding official support to a gitter channel if we do not have members of the project who are going to be actively offering support

@nodejs/collaborators do ya'll have any thoughts on this?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ![node-js](assets/nodejs-black.png) Help
 ====
 
-Need help with Node.js? You've come to the right place! [Create a new issue](https://github.com/nodejs/help/issues/new)!
+Need help with Node.js? You've come to the right place! [Create a new issue](https://github.com/nodejs/help/issues/new)! [Visit out Gitter channel](https://gitter.im/nodejs/help)!
```
Comment (Trott) [inline README.md]: Typo: `out` -> `our`

**Comment by Trott** [comment]
We have a freenode IRC channel that people can use for help questions that we should definitely advertise before this. And there is already a Gitter for the main Node repo, which is the one that should be linked here if we include a Gitter channel at all. No need to create a new one for this repo.

**Comment by Fishrock123** [comment]
We've had this discussion before, and always settled back to IRC.

It's the compromise of accessible, moderateable, etc. (Last I checked, gitter's moderation abilities were terrible.) (If you could GitHub sign into slack...........)

**Comment by ChALkeR** [comment]
We already have a gitter channel for nodejs/node (with ~4000 people) that is used mainly for questions (like #Node.js on freenode).

Is there any reason to split that channel in two for nodejs/node and nodejs/help? What would be the difference between those two?

**Comment by mh-cbon** [comment]
how about just communicating about each channels with a little indication about official support ? In my pparticular case, using gitter over irc was just a matter of simplicity (click and go), in fact.

**Comment by subfuzion** [comment]
I prefer that for people looking to chat we recommend in order (1) the IRC channel and (2) the existing node gitter channel. I don't see any benefit at all to adding yet another gitter channel. For those who don't like the signal-to-noise ratio in the chat channels, that's what this repo is for.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 Join the conversation!
```
Comment (subfuzion) [inline README.md]: We should pick one location or the other, but not leave both.

**Comment by Trott** [comment]
Choices can be bad, especially if you're already confused/frustrated and looking for help. It's one more decision to make when you already feel like you don't know what you're supposed to do. Like, don't give me four places I can try and/or spam. Give me one and make it a good one. In that regard, this repo probably shouldn't direct people elsewhere. Or, at least, if it does, it should do so either in a clear way ("help with X, use this; help with Y, use that") or in a prioritized way ("try X first; if no one helps you after 72 hours, try Y").

**Comment by Trott** [comment]
And, counterpoint to myself: If we have a finite number of supported ways to get help, they should probably all be listed everywhere and in the same order. So, you know, I'm being useless here.

**Comment by Trott** [comment]
Wait a minute. Gitter is already there and has been since September. Oof. I'm going to submit my own PR I think....

**Comment by Trott** [comment]
Inspired by this, I've opened #131. Of course, there's a good chance that my ideas are terrible and this should stay open, because my ideas are often terrible. But I *think* I captured something that approximates the consensus view.

**Comment by lordKnighton** [comment]
don't be to hard on your self, LGTM

**Comment by thefourtheye** [comment]
Are we officially moderating the Gitter room? I personally feel that Gitter has to improve a lot and -1 to this as of now.

Also, if we have too many channels, will we be able to collaborate better?

**Comment by mh-cbon** [comment]
good luck : ) i close this issue and let you focus on the new proposal.

**Comment by Trott** [comment]
@thefourtheye asked:

> Are we officially moderating the Gitter room? 

The Gitter room (associated with the node repo, not talking about the one associated with this repo, which I suspect shouldn't exist really and people should just use the main node repo) is definitely official. I think support for it may be reluctant because the most active folks on core seem to favor IRC, but there's been acknowledgement that people use and like it. I believe @mikeal set up the Gitter room or has administrative access and may have corrections to what I've written or more information that might be relevant.

The only mention I find of it in TSC or CTC meeting notes dates from the io.js days, though: https://github.com/nodejs/node/blob/6fff47ffacfe663efeb0d31ebd700a65bf5521ba/doc/tsc-meetings/io.js/2014-10-29.md

**Comment by Fishrock123** [comment]
We are definitely not officially moderating it with any great frequency. Also, again, gitter had (probably still "has") very poor access to moderation tools.

**Pull request closed without merging**